core/utils.py

```diff
@@ -0,0 +1,38 @@
+import jwt
+import bcrypt
+import re
+
+from django.http import JsonResponse
+
+from my_settings import SECRET, ALGORITHM
+from user.models import User
+
+def login_decorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            token = request.headers.get('Authorization', None)
+            payload = jwt.decode(token, SECRET['secret'], algorithm=ALGORITHM[algorithm])
```

Comment:
```suggestion
            payload = jwt.decode(token, SECRET['secret'], algorithm=ALGORITHM['algorithm'])
```
딕셔너리의 키는 아마도 string일 것입니다.
데코레이터를 아직 테스트 해보시기 전이라 티가 안났을텐데 아마 동작하지 않을거에요 확인부탁드립니다~

Reply:
확인했습니다. 감사합니다.